README.md

```diff
@@ -114,22 +128,42 @@ func main() {
 }
 ```
 
-### Run FileUploader
+**1. Create a test file containing data:**
+
+On Linux systems, you may do this with `dd`:
+```
+dd if=/dev/urandom of=/tmp/testdata bs=2048 count=10
+```
+
+**2. Run FileUploader with the following commands:**
+
 ```sh
-go run file-uploader.go
-2016/08/13 17:03:28 Successfully created mymusic
-2016/08/13 17:03:40 Successfully uploaded golden-oldies.zip of size 16253413
+go mod init example/FileUploader
+go get github.com/minio/minio-go/v7
+go get github.com/minio/minio-go/v7/pkg/credentials
+go run FileUploader.go
+```
+The output resembles the following:
```

Comment:
Not vital, but I'd suggest adding a blank line before and after this one for easier reading when editing this file in the future.

Reply:
Oh yes, you are correct. 👍🏻